### ISSUE TITLE
Incorrect steps to flash, USB connection and running Etcher on Windows

### ISSUE THREAD

**Issue by dnedeltchev** [home-assistant/home-assistant.github.io#16134]
## The problem

Hi all, 

The below instructions are incorrect. 

1. Firstly the device has to be connected to USB *before* powering on the ODROID and running the ums command. 
2. Running Etcher without local admin (on Windows at least, have not tried other OSs) causes Etcher to first prompt for elevation of rights and then it hangs. So on Windows if you have UAC enabled, which is on by default, running Etcher as admin solves the problem. 

At least the above worked for me. Following the steps in the article didn't work. Basically if you follow the steps in the article the ODROID doesn't appear in Windows.

**Comment by frenck** [comment]
Feel free to suggest a change. Every page has a edit link in the top of the page.

**Comment by dnedeltchev** [comment]
Hi Franck, I am not sure what I am doing in GitHub 😊 hence I decided to raise an issue. However, I have now proposed changes to the instructions and after saving, I have an option to Create a pull request. I still don’t know what I am doing 😊 Should I click that button? Many thanks again!D Nedeltchev Sent from Mail for Windows 10 From: Franck NijhofSent: 10 January 2021 22:10To: home-assistant/home-assistant.ioCc: dnedeltchev; AuthorSubject: Re: [home-assistant/home-assistant.io] Incorrect steps to flash, USB connection and running Etcher on Windows (#16134) Feel free to suggest a change. Every page has a edit link in the top of the page.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
